### PR TITLE
Site subtopics

### PR DESCRIPTION
**Pull request by DuttonMa**
This view makes up the lower part of the page when you click on a taxonomy term like 'Motoring' in D7.
To get it working I needed to install the views_contextual_filters_or module.
This view will not actually be displayed anywhere as yet, I will handle that when I get into the 'Articles by Term' view that is coming next !

### CODE REVIEW

File: config/sync/views.view.site_subtopics.yml [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+      max-age: -1
+      contexts:
+        - 'languages:language_content'
+        - 'languages:language_interface'
+        - url
+        - user.permissions
+      tags: {  }
+  block_1:
+    display_plugin: block
+    id: block_1
```
Comment (omahm) [inline config/sync/views.view.site_subtopics.yml]: Use a more descriptive machine name.

File: config/sync/views.view.site_subtopics.yml [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+      query:
+        type: views_query
+        options:
+          disable_sql_rewrite: false
+          distinct: false
+          replica: false
+          query_comment: ''
+          query_tags: {  }
+          contextual_filters_or: 1
+      exposed_form:
```
Comment (johangant) [inline config/sync/views.view.site_subtopics.yml]: How does the exposed filter block fit into the use case for this view? Or is this boilerplate output that I'd misinterpreted? 😛

File: config/sync/views.view.site_subtopics.yml (hunk 1 above)
Comment (DuttonMa) [reply config/sync/views.view.site_subtopics.yml]: Yes, have done that now. tbh I hadn't actually decided yet whether to use a block or an 'embed', that will come in phase 2.

File: config/sync/views.view.site_subtopics.yml (hunk 2 above)
Comment (DuttonMa) [reply config/sync/views.view.site_subtopics.yml]: There are two contextual filters for this view, both of which pull their import from the URL:
- 'parent' which checks the immediate parent of this term in the taxonomy hierarchy
- 'supplementary parent' which checks whether this term has other 'parents' in addition to the term above it in the hierarchy

'Views contextual filters OR' allows rows to be displayed by this view if they match either of the contextual filters above.

Comment (johangant) [reply config/sync/views.view.site_subtopics.yml]: 👍

**Pull request merged**